server/src/main/java/org/opensearch/action/support/clustermanager/TransportClusterManagerNodeAction.java

```diff
@@ -117,13 +117,24 @@ protected TransportClusterManagerNodeAction(
 
     protected abstract Response read(StreamInput in) throws IOException;
 
-    protected abstract void masterOperation(Request request, ClusterState state, ActionListener<Response> listener) throws Exception;
```

Comment:
@andrross Do you think we'd better add a comment in the code to remind adding back the `abstract` keyword after removing the deprecated method?

Reply:
Can we follow this up right after this is merged with a PR on main that removes the master methods and makes the methods abstract? If for some reason we can't do that then a comment is fine.

Reply:
I'm not sure we are going to remove the deprecated methods in version 3.0, which is the code in `main` branch, but I will write it down in to remind in issue https://github.com/opensearch-project/OpenSearch/issues/3542 or a new issue.